api/key.go

```diff
@@ -11,7 +11,8 @@ import (
 
 // Key provides a key for signing authentication tokens.
 type Key struct {
-	bytes []byte
+	bytes    []byte
+	disabled bool
```

Comment:
I have another idea of how to implement this. Since

- we know at startup whether we want authentication, and
- clients are agnostic about token format

we could create a separate type:

```
type AlwaysValid struct{}

func (AlwaysValid) NewToken() (string, error) { return "valid", nil }
func (AlwaysValid) Valid(*http.Request) bool  { return true }
```

This keeps conditional logic out of the `Key` implementation. I think it better expresses the intention.

Reply:
implemented -- had to create another interface to pass from main -- does this make sense?

Reply:
Yes; I forgot that no interface yet existed for this.